cachier/cores/pickle.py

```diff
@@ -19,19 +19,11 @@
 # Alternative:  https://github.com/WoLpH/portalocker
 
 from .base import _BaseCore
+from ..config import _Type_HashFunc
 
 
 class _PickleCore(_BaseCore):
-    """The pickle core class for cachier.
-
-    Parameters
-    ----------
-    pickle_reload : bool, optional
-        See core.cachier() documentation.
-    cache_dir : str, optional.
-        See core.cachier() documentation.
-
```

Comment:
Why are we dropping parameter documentation of the pickle core? Do we not get, as devs, to have some kind of memory and documentation about these? Isn't separate files, in the very least, unique to the pickle core?

Reply:
It was not compete so I saw two options drop or remove and as other classes doesn't have it either...